src/backend/core/migrations/0020_alter_user_language.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 5.1.7 on 2025-03-17 14:00
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("core", "0019_alter_user_language_default_to_null"),
+    ]
+
+    operations = [
+        migrations.AlterField(
```

Comment:
This is noop, I would not make a migration for this

Reply:
Thanks